### DISCUSSION TITLE
Special Request

### DISCUSSION THREAD

**Discussion by ghost** [TeamUltroid/Ultroid#157] [General]
Dear owners, @xditya I wanted to make Ultroid-X Userbot But What Are the terms and conditions to create

**Comment by xditya** [comment]
Read the [license](https://github.com/TeamUltroid/Ultroid/blob/main/LICENSE).

**Reply by ghost** [reply]
Thanks I am creating with giving all credits + co authoring all devs of Ultroid so Kindly do not give DMCA. @xditya

**Reply by xditya** [reply]
As long as you follow the license.
